starter-code/src/memory.js

```diff
@@ -1,15 +1,47 @@
-// var MemoryGame = function (cards) {
-//   this.cards = cards;
-// };
+'use strict'
 
-// MemoryGame.prototype.shuffleCard = function (cardsArr) {
 
-// };
+var MemoryGame = function (cards) {
+ this.cards = cards;
+ this.pickedCards = [];
+ this.pairsClicked = 0;
+ this.pairsGuessed = 0;
+ 
+};
 
-// MemoryGame.prototype.checkIfPair = function (firstCard, secondCard) {
+MemoryGame.prototype.shuffleCard = function (cardsArr) {
```

Comment:
You need to invoke this function in some point, otherways the cards are not shuffled. You can call it in line 9:
```
var MemoryGame = function (cards) {
  this.cards = cards;
  this.pickedCards  = [];
  this.pairsClicked = 0;
  this.pairsGuessed = 0;
  this.cards = this.shuffleCard(cards);
};
```